ballerina-test/src/test/resources/openshift/hello_world_oc.bal

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This has to be 2020

Reply:
https://github.com/ballerina-platform/ballerina-distribution/pull/444/commits/96b01c79136f64c90671e0b7151493c75c047704